src/guides/v2.3/config-guide/cron/custom-cron-tut.md

```diff
@@ -174,6 +174,19 @@ Create `crontab.xml` as follows in the `/var/www/html/magento2/app/code/Magento/
 
 The preceding `crontab.xml` runs the `Magento/SampleMinimal/Cron/Test.php` class once per minute, resulting in a row being added to the `cron_schedule` table.
 
+In order to make the cron schedule configurable from admin panel, you may use the configuration path of your system configuration field.
+
+```xml
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
+        <job name="custom_cronjob" instance="Magento\SampleMinimal\Cron\Test" method="execute">
+            <config_path>system/config/path</config_path>
```

Comment:
Please, add a small hint regarding what `<config_path>` is. Like 

```
... where config_path is a system configuration path defined in `etc/adminhtml/system.xml` of a module. 
```

Thank you!

Reply:
Thank you, updated.

Reply:
Thank you!